cmsplugin_filer_link/forms.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django import forms
+
+from djangocms_attributes_field.widgets import AttributesWidget
+
+from .models import FilerLinkPlugin
+
+
+class FilerLinkForm(forms.ModelForm):
+
+    class Meta:
+        model = FilerLinkPlugin
+        exclude = []
+
+    def __init__(self, *args, **kwargs):
+        super(FilerLinkForm, self).__init__(*args, **kwargs)
+        self.fields['link_attributes'].widget = AttributesWidget(val_attrs={'style': 'width: 500px!important'})
```

Comment:
maybe something like ``width: 100%`` could be set as the default of the widget... avoiding the need to redifine it all over the place. ``500px`` is probably not so good for responsiveness.